Dockerfile

```diff
@@ -5,9 +5,12 @@ COPY / .
 RUN make get \
 && make build
 
-FROM chromedp/headless-shell:latest
```

Comment:
This change seems unrelated, could you please remove it?

Reply:
Sorry about this. This was never intended to be pushed in this repo. It is an internal constraint on our side.